File: backend/backend-main/models/News.js
const mongoose = require('mongoose');


const Reclamation = mongoose.model('News',{
    title: {
        type:String,
        require : true
    },
    description: {
        type:String,
        require : true
    },
    img: {
        type: String,
        // require : true
    },
    date: {
        type: DateTime,
        // require : true
    },
    id_user: {
        type:integer,
    },
    id_admin : {
        type:integer,
    }
})

module.exports = News;

File: Ecocitoyen_frontend/app/src/main/java/com/example/ecocitoyen_frontend/Models/News.kt
package com.example.ecocitoyen_frontend.Models
 class News(
    val id: Int,
    val title: String,
    val description: String,
    val image: String,
    val date: String,
    val id_user: Int
)
 {
     override fun toString(): String {
         return "News(id=$id, title='$title', description='$description', image='$image', date='$date', id_user=$id_user)"
     }
 }
